### ISSUE TITLE
Feature Request: "Other"

### ISSUE THREAD

**Issue by Frank-Marion** [8manos/wc-city-select#22]
I don't yet have quite have the skills to add this. City lists can be incomplete, or perhaps have misspellings. It would be useful to have an "other" option that permits the user to enter something not in the select menu.

**Comment by Mantish** [comment]
My guess is that it shouldn't be too difficult, as select2 support something like this: https://select2.org/tagging
@Frank-Marion Would you like to try implementing it?

**Comment by Frank-Marion** [comment]
No, this one is currently beyond my knowledge and abilities, I think. Otherwise, I would have addressed it myself.

**Comment by tsbohc** [comment]
I won't submit a PR, but will leave this here in case it helps someone.

All you need is to add `tags: true` here:

https://github.com/8manos/wc-city-select/blob/2b79a1da79cd33c8f47d36f5eeb66531ac9fddbc/assets/js/city-select.js#L64-L67

Like so (note the comma on the previous line):

```
        var select2_args = $.extend({
          placeholderOption: 'first',
          width: '100%',
          tags: true
        }, getEnhancedSelectFormatString() );
```

And don't forget to hard refresh the page (Ctrl+Shift+R).
